reactioncommerce/reaction
# (fix) Properly return 0 when no tax items are present

Resolves #4669 
Impact: **major*  
Type: **bugfix**

## Issue
If `item.tax` did not exist the `getTaxTotal` was returning `NaN` which would show up in cart totals. This was a result of my previous PR fixing custom taxes.

## Solution
Properly return 0 when these don't exist


## Testing
1. Start system with fresh db and don't enable taxes
1. Attempt to checkout
1. Observe that that the cart total is always calculated correctly

More detail for what each of these sections should include are available in our [Contributing Docs](https://docs.reactioncommerce.com/reaction-docs/master/contributing-to-reaction)
